Library/Formula/cdrtools.rb

```diff
@@ -37,6 +37,14 @@ def install
     (lib/"profiled").rmtree
     man5.rmtree
   end
+
+  test do
+    system "#{bin}/cdrecord", "-version"
+    system "#{bin}/cdrecord", "-scanbus"
+    system "#{bin}/mkisofs", "-version"
+    system "#{bin}/cdda2wav", "-version"
+    system "#{bin}/readcd", "-version"
```

Comment:
Just remove the `-version` calls; the `-scanbus` is enough. Thanks for the test though!